standalone-tools/wss-scan.config

```diff
@@ -0,0 +1,4 @@
+emailid=
+baseDirPath=$(pwd)
+gitBasePath=https://github.com/opendistro-for-elasticsearch/
+gitRepos=alerting,simple-ingest-transformation-utility-pipeline,opendistro-build,security,performance-analyzer-rca,anomaly-detection,index-management,security-kibana-plugin,sql,job-scheduler,kibana-reports,notifications,performance-analyzer,index-management-kibana-plugin,alerting-kibana-plugin,common-utils,k-NN,anomaly-detection-kibana-plugin,perftop,kibana-notebooks,sql-workbench,sql-cli,sql-jdbc,sql-odbc,piped-processing-language
```

Comment:
This is a very bad way to store the information on github repos, especially when you can do an API call to directly retrieve the existing repos you want under opendistr-for-elasticsearch org.

Considering not hardcoding these information since ppl can add or remove repos at any given time

Reply:
This script will only be used by our team. And the script is designed for a scheduled run, so the changes to the repos will be less often. If in case our team wants to run the script on our desktop, this config will provide a easy way to scan only the repos we need.

Reply:
This is not about local vs remote run, I am specifically talking about not to hardcode gitRepos parameters as default, but use a dynamic retrieval from API call.

I am fine with specific local runs with hardcoding.

Reply:
The whitesource can be used only to scan Open Source Repos. We shouldnt scan private repos, I will make the change to move to API once we have the tags informing the Repos which are private.